ztools/telnet/log.go

```diff
@@ -0,0 +1,13 @@
+/*
+ * ZGrab Copyright 2015 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
```

Comment:
Can you add the warranty clause here? It should be in the other files, I went through and added it to all files a couple weeks ago.